src/box/lua/config/utils/schema.lua

```diff
@@ -1492,6 +1492,88 @@ end
 
 -- }}} <schema object>:pairs()
 
+-- {{{ <schema object>:jsonschema()
+
+local json_scalars = {
+    string = {jsonschema = {type = 'string'}},
+    number = {jsonschema = {type = 'number'}},
+    ['string, number'] = {jsonschema = {type = {'string', 'number'}}},
+    ['number, string'] = {jsonschema = {type = {'string', 'number'}}},
+    integer = {jsonschema = {type = 'integer'}},
+    boolean = {jsonschema = {type = 'boolean'}},
+    any = {jsonschema = {}}
+}
```

Comment:
Let's just include it into the scalar definitions (`scalars` defined at line 170-274).

Reply:
Done.